### ISSUE TITLE
Fullscreen broken on iPhone (and possibly other mobile devices)

### ISSUE THREAD

**Issue by RubenVerborgh** [shower/shower#234]
Fullscreen does not work anymore on iPhone, and possibly other mobile devices. For instance, the home page http://shwr.me/ breaks, with various effects such as:

![image](https://cloud.githubusercontent.com/assets/675313/12823853/b0c13302-cb6e-11e5-89f1-83e9c0b2b941.png)
![image](https://cloud.githubusercontent.com/assets/675313/12823867/bbb440ce-cb6e-11e5-92ba-88330c74e1a9.png)

It worked nicely with previous versions, e.g., shower/shwr.me@1b8929e9b2155baa444b7b3516de6b0dd4d3db2e, but broke when 
```HTML
<meta name="viewport" content="width=792, user-scalable=no">
```
was replaced by
```HTML
<meta name="viewport" content="width=device-width, initial-scale=1">
```

However, more seems to be up, because `user-scalable=no` doesn't give the desired effect anymore (it zooms in way too much). The most acceptable solution that I found was:
```HTML
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
```
(both initial and maximum are needed).

Yet this does not prevent a vertical positioning error from occurring every now and then:
![image](https://cloud.githubusercontent.com/assets/675313/12824010/6018fe02-cb6f-11e5-805d-f0a9fe0fb612.png)

However, the vertical positioning gets fixed as soon as you navigate to the next slide:

![image](https://cloud.githubusercontent.com/assets/675313/12824048/8d201e3a-cb6f-11e5-8adf-0659bbfe51f1.png)

Finally giving the expected result—except for the browser bars that won't disappear, but that's another issue (#224).

**Comment by RubenVerborgh** [comment]
On closer investigation, the top positioning problem in the second to last screenshot (with initial-scale and maximal-scale set) seems to be caused by `document.body.scrollTop === 5`, whereas `document.body.scrollTop === 74` on subsequent slides.

**Comment by RubenVerborgh** [comment]
Actually, [iOS 9 seems to require an additional `shrink-to-fit` key](http://kihlstrom.com/2015/shrink-to-fit-no-fixes-zoom-problem-in-ios-9/):

```
<meta name="viewport" content="widh=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no">
```

**Comment by pepelsbey** [comment]
Fixed in [ribbon](https://github.com/shower/ribbon/commit/f8c2454272f73afafe614ab6e023e3485cb4d0b6) and [material](https://github.com/shower/material/commit/5dbe4a03b958b051a1ce1ad29aba3d667d013962) themes. Thank you again!

**Comment by RubenVerborgh** [comment]
@pepelsbey Not sure if everything is fixed by shower/ribbon#10. It solves the weird horizontal scrolling offsets, but a vertical shift can still occur:

![](https://cloud.githubusercontent.com/assets/675313/12824010/6018fe02-cb6f-11e5-805d-f0a9fe0fb612.png)

This might be related to #224 though, as it doesn't seem like a coincidence that the vertical offset is equal to the height of the address bar.
